layouts/community/ortho_4x12/buswerks/keymap.c

```diff
@@ -0,0 +1,176 @@
+),
+
+[_LOWER] = LAYOUT_ortho_4x12( \
+  // .--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------.
+       KC_TILD, _______, _______, _______, _______, _______, _______, KC_7,    KC_8,    KC_9,    KC_MINS, KC_DEL, \
+  // |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+       _______, KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC, _______, KC_4,    KC_5,    KC_6,    KC_PLUS, KC_PIPE, \
+  // |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+       _______, KC_CIRC, KC_ASTR, KC_UNDS, KC_EQL,  KC_AMPR, _______, KC_1,    KC_2,    KC_3,    KC_SLSH, _______, \
+  // |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+       _______, _______, _______, _______, _______, _______, _______, KC_0,    _______, _______, _______, _______ \
+  // '--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------'
+),
+
+[_RAISE] = LAYOUT_ortho_4x12( \
+  // .--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------.
+       KC_GRV,  _______, _______, KC_LPRN, KC_RPRN, _______, KC_MNXT, KC_VOLD, KC_VOLU, KC_MPLY, _______, KC_DEL, \
+  // |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+       _______, _______, _______, KC_LCBR, KC_RCBR, _______, KC_LEFT, KC_DOWN, KC_UP  , KC_RGHT, _______, KC_BSLS, \
+  // |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+       _______, _______, _______, KC_LBRC, KC_RBRC, _______, KC_HOME, KC_PGDN, KC_PGUP, KC_END,  _______, _______, \
+  // |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+       _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______ \
+  // '--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------'
+),
+
+[_ADJUST] =  LAYOUT_ortho_4x12( \
+  KC_F11,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F12, \
+  _______, _______, RESET,   AU_ON,   AU_OFF,  AG_NORM, AG_SWAP, QWERTY,  COLEMAK, DVORAK,  _______, _______, \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______ \
+)
+
+
+};
+
+#ifdef AUDIO_ENABLE
+float tone_qwerty[][2]     = SONG(QWERTY_SOUND);
+float tone_dvorak[][2]     = SONG(DVORAK_SOUND);
+float tone_colemak[][2]    = SONG(COLEMAK_SOUND);
+#endif
+
+void persistent_default_layer_set(uint16_t default_layer) {
```

Comment:
This function is depreciated, Specifically, it was rolled into the core code and made more "friendly" and functional. 

Please remove the function.